cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfOperationsCreator.java

```diff
@@ -10,6 +10,9 @@
 
 public class CfOperationsCreator {
 
+    private static final String CF_CONTROL_USER = "CF_CONTROL_USER";
+    private static final String CF_CONTROL_PASSWORD = "CF_CONTROL_PASSWORD";
+
     public static DefaultCloudFoundryOperations createCfOperations(
```

Comment:
A documentation explaining what the constructor does would be nice, since its behaviour has changed by now.